RUFAS/routines/manure/pen_manure/manure_manager_pen.py

```diff
@@ -130,6 +128,6 @@ def exposed_manure_surface_area_from_pen_type(self) -> float:
 
         exposed_manure_surface_area = exposed_manure_surface_area_by_pen_type[self.pen_type]
 
-        if "LacCow" in self.classes_in_pen:
+        if AnimalCombination.LAC_COW == self.animal_combination:
```

Comment:
Reversing the logic here might read better, but it's the same functionality. Just thinking ahead in case there's more branching logic in the future, since checking `self.animal_combination` against lists might come up.


```suggestion
        if self.animal_combination == AnimalCombination.LAC_COW:
```